src/overlays/actors/ovl_En_Daiku/z_en_daiku.c

```diff
@@ -207,9 +207,9 @@ void func_809438F8(EnDaiku* this, PlayState* play) {
 
         sq = sqrtf(SQ(this->actor.world.pos.x - this->unk_26C.x) + SQ(this->actor.world.pos.z - this->unk_26C.z));
```

Comment:
Not you, but `sq` is incorrect here since the sqrt is taken, meaning `SQ(2.0f)` should just be `4.0f` below

Reply:
Renamed the variables as well.